packages/cli/src/cmds/validator/options.ts

```diff
@@ -127,6 +128,14 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
   ...logOptions,
   ...keymanagerOptions,
 
+  broadcastValidation: {
+    // TODO: flag hidden till validations fully implemented
+    hidden: true,
+    description: "Broadcast validation to be requested before publishing the block",
+    defaultDescription: defaultOptions.broadcastValidation,
+    type: "string",
+  },
```

Comment:
I don't think this needs to be implemented on the validator's side? I don't think we ever want extra validations to run before publishing its blocks

Reply:
hmm.... default can be set to none whch is what its currently set, but some builder using lodestar validator to publish can provide this flag

Reply:
imo since this feature is specifically for relay client diversification, we shouldn't touch validator code here, and it makes it a safer PR since we know there are no validator changes.
A builder using lodestar to publish should enable broadcast validation on its own part of the stack, not in the validator.

Reply:
how about those using the DVT where dvt only sits in the middleware and the distributed validators do want to validate the block... but i guess we can leave that out of scope for now in the least. 

so just remove the broadcastValidation configuration option or not even move validator calls to produce[Block|BlindedBlock] to v2?